microprot/scripts/pdb_search.py

```diff
@@ -388,3 +391,156 @@ def report_uncovered_subsequences(hits, query, min_subseq_len=40):
 
     return [u for u in uncovered if (u['sequence'] != '') &
                                     (len(u['sequence']) >= min_subseq_len)]
+
+
+def get_q_id(hit):
+    """ Returns the query ID for a hit.
+
+    Parameters
+    ----------
+    A hit parsed from an HHsearch output file, i.e. dict with at least the key
```

Comment:
I feel that the description isn't very clear. By looking at the context `hit['alignment']['Q Consensus']['start']` I began to understand :) Why this data structure is so complicated?